synfig-studio/src/gui/app.cpp

```diff
@@ -1058,14 +1058,9 @@ DEFINE_ACTION("help-about",     Gtk::StockID("synfig-about"))
 DEFINE_ACTION("keyframe-properties", _("Properties"))
 
 // actions: move to tab
-DEFINE_ACTION("move-to-tab-1",  _("Move to Tab 1"))
-DEFINE_ACTION("move-to-tab-2",  _("Move to Tab 2"))
-DEFINE_ACTION("move-to-tab-3",  _("Move to Tab 3"))
-DEFINE_ACTION("move-to-tab-4",  _("Move to Tab 4"))
-DEFINE_ACTION("move-to-tab-5",  _("Move to Tab 5"))
-DEFINE_ACTION("move-to-tab-6",  _("Move to Tab 6"))
-DEFINE_ACTION("move-to-tab-7",  _("Move to Tab 7"))
-DEFINE_ACTION("move-to-tab-8",  _("Move to Tab 8"))
+for (int i = 1; i <= 8; ++i) {
```

Comment:
please use:
```
for () {
	...
} <- on the new line
```

Reply:
alright, and by the way did you guys set the new rules for indentation that was being talked about on the forums yet?

Reply:
I am not sure. 

Anyway I am trying to copy the style from the previous code, in this case the new code will look the same.